From 1658083130e750d1f194460fcf1fd81d5df9bc48 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 21 May 2018 08:55:57 +1000
Subject: [PATCH libinput] configure.ac: correct the version number

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 197dbe42..7201163a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,8 +1,8 @@
 AC_PREREQ([2.64])
 
 m4_define([libinput_major_version], [1])
-m4_define([libinput_minor_version], [8])
-m4_define([libinput_micro_version], [0])
+m4_define([libinput_minor_version], [10])
+m4_define([libinput_micro_version], [7])
 m4_define([libinput_version],
           [libinput_major_version.libinput_minor_version.libinput_micro_version])
 
-- 
2.14.3

